src/elements/selection-expansion-panel/selection-expansion-panel.stories.ts

```diff
@@ -45,6 +45,13 @@ const borderless: InputType = {
   },
 };
 
+const size: InputType = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['m', 's'],
+};
```

Comment:
should we have a group indicating that it isn't a property of the expansion panel itself?